### ISSUE TITLE
Performance problem

### ISSUE THREAD

**Issue by zkcys001** [HobbitLong/SupContrast#52]
Thank you for the great work!

I have been training the network with your code. In the Pretraining stage, I use 8 GPU to train in the cifar10 task and use this command:
python main_supcon.py --batch_size 1024 \
  --learning_rate 0.5 \
  --temp 0.1 \
  --cosine

Then, in the Linear evaluation stage, I also adopt the following code to train:

python main_linear.py --batch_size 512 \
  --learning_rate 5 \
  --ckpt /path/to/model.pth

But the performance is 95.58. But your report performance is 96.0. Could you check it is normal?
![image](https://user-images.githubusercontent.com/20808715/102705348-cb306580-42c1-11eb-9e70-d852d0f7872b.png)
Thanks~

Kecheng

**Comment by HobbitLong** [comment]
Hi, @zkcys001 ,

I got two runs, one giving 95.9% and the other 96.0%. Your number might fall into the variation range, but I would try different learning rates in the linear evaluation stage to double check things.

**Comment by Hibb-bb** [comment]
Hi @HobbitLong , may I ask which checkpoint did you use?
I also get a similar accuracy of 95.56% using the checkpoint of the last epoch.
The previous checkpoints like 350, 600  showed worse than 95.56%
